src/coreclr/gc/gcpriv.h

```diff
@@ -131,7 +131,7 @@ inline void FATAL_GC_ERROR()
 // This means any empty regions can be freely used for any generation. For
 // Server GC we will balance regions between heaps.
 // For now disable regions for StandAlone GC, NativeAOT and MacOS builds
-#if defined (HOST_64BIT) && !defined (BUILD_AS_STANDALONE) && !defined(__APPLE__)
+#if defined (BUILD_AS_STANDALONE) && defined (HOST_64BIT) && !defined(__APPLE__)
```

Comment:
This looks like a temporary change to flip clrgc.dll to regions...

Reply:
ooh thanks! forgot to flip it back.